phpBB/styles/prosilver/template/pm.js

```diff
@@ -1,80 +1,31 @@
-var modal = document.getElementById('pm-smile-model');
-var btn = document.getElementById("pm-smile-button");
-var span = document.getElementsByClassName("pm-button-close")[0];
-
-var to_modal = document.getElementById('pm-to-model');
-var to_btn = document.getElementById("pm-to-button");
-var to_span = document.getElementsByClassName("pm-to-button-close")[0];
-
-var bb_modal = document.getElementById('pm-bbcode-model');
-var bb_btn = document.getElementById("pm-bbcode-button");
-var bb_span = document.getElementsByClassName("pm-to-button-close")[0];
-
-/**
- * When the user clicks the button, open the modal
- */
-btn.onclick = function() {
-	modal.style.display = "block";
-}
-
 /**
- * When the user clicks on <span> (x), close the modal
+ * Open Pm smiley popup
  */
-span.onclick = function() {
-	modal.style.display = "none";
-}
+$(document).ready(function(){
+	$('#pm-smile-button').click(function(){
+		var $alert = phpbb.alert("Pm To: Box", $('#pm-smile-model').html());
```

Comment:
Use a data element to call these not ids data-modal=pm-smile and then get the data attr to apply the click on.